src/main/java/org/openhab/binding/zwave/handler/ZWaveControllerHandler.java

```diff
@@ -685,7 +693,7 @@ public ZWaveNode getNode(int node) {
      * This will not wait for the transaction response.
      *
      * @param transaction
-     *            the {@link ZWaveCommandClassTransactionPayload} message to send.
+     *                        the {@link ZWaveCommandClassTransactionPayload} message to send.
```

Comment:
I'm wondering why this changed - did you run the project formatter on this or are you using a different formatter?

Reply:
This was a side effect of the project formatter - the OH eclipse build has a Save Action of Format entire file.  Do you want me to revert this and/or put it in a separate PR?

Reply:
Yes, this is fine if you have used the ESH formatter? It just seems to have formatted differently to my formatter, but maybe the formatter has been updated since I last installed my system.

Reply:
Yes, I have verified it is the ESH formatter in Eclipse